src/index.html

```diff
@@ -11,12 +11,34 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img src="./images/logo.png"
+          alt="logo"
+          class="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_link is-active on-hover"><a href="index.html" class="nav_item">Apple</a></li>
+          <li class="nav_link is-active on-hover"><a href="index.html" class="nav_item">Samsung</a></li>
```

Comment:
Remove these classes for all links, not for apple
```suggestion
          <li class="nav_link">
            <a href="index.html" class="nav_item">Samsung</a>
          </li>
```